src/persist-client/src/impl/state.rs

```diff
@@ -26,13 +26,30 @@ use crate::ShardId;
 
 #[derive(Clone, Debug, PartialEq)]
 pub struct ReadCapability<T> {
+    // WIP: This is not used anywhere. We should think hard about when/if we
```

Comment:
this is how i imagine gc of unused blobs will work, so let's keep it for now. we might want to rename this struct to SeqNoCapability?